### ISSUE TITLE
[Request] Add BatchNorm Parameter to Conv layers

### ISSUE THREAD

**Issue by pietz** [fchollet/keras#5983]
Since it's pretty much standard these days to have BatchNorm after a Conv layer, why not add a parameter to do it in one line.
